diff --git a/default.nix b/default.nix
index 69c1e56..00dc388 100644
--- a/default.nix
+++ b/default.nix
@@ -1,5 +1,5 @@
-{ holoport ? { outPath = ./.; revCount = 0; shortRev = "holoportos-testnet-dev-18.09"; }
-, nixpkgs ? { outPath = fetchTarball "https://github.com/Holo-Host/nixpkgs-channels/archive/nixos-18.09.tar.gz";
+{ holoport ? { outPath = ./.; revCount = 0; shortRev = "master"; }
+, nixpkgs ? { outPath = fetchTarball "https://github.com/NixOS/nixpkgs-channels/archive/nixos-18.09.tar.gz";
               revCount = 0; shortRev = "latest"; }
 , system ? builtins.currentSystem, nixpkgsArgs ? { config = { allowUnfree = true; inHydra = true; }; }
 }:
@@ -35,7 +35,7 @@ rec {
   };
 
   channels.holoport-testnet = pkgs.releaseTools.makeSourceTarball {
-    name = "holoport-testnet-dev-18_09-channel";
+    name = "holoport-testnet-channel";
     src = holoport;
     version = lib.fileContents ./.version;
     versionSuffix = ".${toString holoport.revCount}.${holoport.shortRev}";
@@ -54,7 +54,7 @@ rec {
   };
 
   tested = lib.hydraJob (pkgs.releaseTools.aggregate {
-    name = "nixos-${channels.nixpkgs.version}+holoport-testnet-dev-${channels.holoport-testnet.version}";
+    name = "nixos-${channels.nixpkgs.version}+holoport-testnet${channels.holoport-testnet.version}";
     meta = {
       description = "Release-critical builds for holoportOS";
     };
